src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const totalCost = days * dailyRate;
+  const smallDiscount = 20;
+  const discount = 50;
+  let cost = 0;
+
+  if (days < 3) {
+    return totalCost;
+  }
+
+  if (days < 7) {
+    cost = totalCost - smallDiscount;
+
+    return cost;
+  }
+
+  cost = totalCost - discount;
+
+  return cost;
```

Comment:
Prefer `if` with `return` over `if else` to simplify later conditions. You don't need to use `else` after `if` with `return` because the code after it won't be executed anyway. This will make the code more concise and easier to read.